packages/swap/src/routes/quote.ts

```diff
@@ -226,7 +226,9 @@ const quote = (io: Server) => {
         const message =
           err instanceof Error ? err.message : 'unknown error (possibly no liquidity)';
 
-        logger.error('error while collecting quotes:', err);
+        if (!message.includes('InsufficientLiquidity')) {
+          logger.error('error while collecting quotes:', err);
+        }
 
         // DEPRECATED(1.3): remove `error`
         res.status(500).json({ message, error: message });
```

Comment:
Also I think returning 500 would cause the result to be logged as error, but not sure